### ISSUE TITLE
Update annual revenue for FY2018

### ISSUE THREAD

**Issue by mentastc** [ONRR/doi-extractives-data#3478]
Update annual revenue for FY2018

- [x] Check file to ensure it adheres to the new file format

- [x] Check file to ensure there are no obvious data issues

- [x] Update Data Download

**Comment by mentastc** [comment]
Attached is the file to be placed for download. 
[FY18 Federal Revenues (1).xlsx](https://github.com/ONRR/doi-extractives-data/files/2676354/FY18.Federal.Revenues.1.xlsx)

**Comment by brentryanjohnson** [comment]
Since we've consolidated the revenue data into a single file (instead of the previous three), we need to modify the language. Eventually, we'll need to rewrite the full documentation on the page, since it assumes the separation of data files.

For now, I've rewrote the intro copy:

> We offer revenue data files for both calendar year and fiscal year. Calendar year revenue data is split into three separate files (onshore, offshore, and revenues not associated with a lease). Calendar year data is available for calendar years 2006 through 2017. Fiscal year data is available in a single file, for fiscal years 2006–2018. They are all accounting year data.

I've changed the microcopy for the link as well:

![link language change for revenue data reading onshore and offshore revenue dataset for fiscal year 2006 to 2018](https://user-images.githubusercontent.com/32855580/49954318-0b7bd280-feb5-11e8-926c-12f8f773b4c2.png)

Let me know what you think of these changes, @mentastc.
